corehq/apps/export/views/utils.py

```diff
@@ -9,6 +9,7 @@
 from django.views.generic import View
 
 from memoized import memoized
+from corehq.apps.hqwebapp.decorators import use_bootstrap5
```

Comment:
nit: can you move this import down to other `corehq` imports?

Reply:
Yes, moved in [600f240](https://github.com/dimagi/commcare-hq/pull/35140/commits/600f240e25a1d34a4b226f88a9465826534c437d)